PARQUET-1031: Fix spelling errors, whitespace, GitHub urls

commanderofthegrey opened the pull request:
rebased pull request https://github.com/apache/parquet-format/pull/39, fixed minor spelling mistake and the travis-ci URLs (which also pointed to the Parquet/parquet-format one).

@Mistobaan please  let me know if you would like to reclaim the original pull request.

nandorKollar commented:
@commanderofthegrey it seems that the link for "record shredding and assembly algorithm" doesn't point to the intended location, maybe the file was removed meanwhile? Should we remove the link since it point so a meaningless location?

rdblue commented on README.md:
```diff
@@ -146,90 +146,91 @@ documented in
+ 1. encoded values
 
-The value of `uncompressed_page_size` specified in the header is for all 3 pieces combined.
+The size of the page specified in the header is for all the 3 pieces combined.
```
This should keep the reference to `uncompressed_page_size`. It is useful to know that.

rdblue commented:
Could you rebase and fix the one thing I pointed out? I'll merge this and get it into the 2.3.2 release if you do. Thank you!

commanderofthegrey commented:
Hi @rdblue, @nandorKollar thanks for the reviews! 
I've rebased and reverted the change to  `uncompressed_page_size`.
Please let me know if there are any further changes needed.

Thanks,
/cmd

rdblue commented:
Thanks, @commanderofthegrey! I merged the changes.
